spec/order_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'order'
+
+describe Order do
+  let(:restaurant) {double (:restaurant), :menu => menu, :check_inclusion? => true, :get_price => 10}
+  let(:invalid_choice) {double (:restaurant), :menu => menu, :check_inclusion? => false}
+  let(:menu) {double (:menu), items: { pizza: 10, lasagne: 8, omlette: 4 }}
+
+  describe '#add' do
+
+    it 'raises an error if the food_item is invalid' do
+      expect{subject.add invalid_choice, :chips}.to raise_error "Invalid menu selection. Please try again."
```

Comment:
Line is too long. [107/80]<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.